JsonSchema.Data.Tests/Suite/Suite.cs

```diff
@@ -43,10 +55,7 @@ public static IEnumerable<TestCaseData> TestCases()
 		{
 			var shortFileName = System.IO.Path.GetFileNameWithoutExtension(fileName);
 			var contents = File.ReadAllText(fileName);
-			var collections = JsonSerializer.Deserialize<List<TestCollection>>(contents, new JsonSerializerOptions(DataTestsSerializerContext.OptionsManager.SerializerOptions)
-			{
-				PropertyNameCaseInsensitive = true
-			});
+			var collections = JsonSerializer.Deserialize<List<TestCollection>>(contents, _serializerOptions);
```

Comment:
Similarly, this call should be able to just pass DataTestsSerializerContext.Default.ListTestCollection, right?

Reply:
I wanted to be able to set the options here.